apps/webapp/src/app/meeting-add/meeting-edit.tsx

```diff
@@ -70,6 +71,7 @@ export const MeetingEdit = () => {
   const [error, setError] = useState<string>();
   const [saveSnackbar, setSaveSnackbar] = useState<boolean>(false);
   const [change$] = useState(new ReplaySubject());
+  document.title = useDocumentTitle(meeting);
```

Comment:
hmm, a co dodaje ten hook ponad:
`document.title = meeting?.title ?? 'Cebulendarz'`

Reply:
reużywalność (oszczędnośc 5lini w komponencie)

Reply:
chcemy do tego jeszcze wracać czy już to zostawiamy? ;)